README.md

```diff
@@ -1,233 +1,249 @@
+| REDIS_LOCAL                      | Boolean to determine if Redis runs locally or not                                                                                                                                       |No|
+| TENANT_ID                        | Directory unique ID assigned to our Azure AD tenant. Represents the organisation that owns and manages the Azure AD instance.                                                           |No|
+| DATA_MANAGEMENT_AZ_API           | Used as part of the `scope` parameter when requesting a token from Azure. Used for service-to-service communication with the pip-data-management service                                |No|
+| ACCOUNT_MANAGEMENT_AZ_API        | Used as part of the `scope` parameter when requesting a token from Azure. Used for service-to-service communication with the pip-account-management service                             |No|
+| SUBSCRIPTION_MANAGEMENT_AZ_API   | Used as part of the `scope` parameter when requesting a token from Azure. Used for service-to-service communication with the pip-subscription-management service                        |No|
+| ENABLE_CFT                       | Boolean determining whether CFT IDAM login is possible (defaults to false for local)                                                                                                    |No|
+| CFT_REJECTED_ROLES_REGEX         | Allows you to override the rejected roles regex for CFT                                                                                                                                 |No|
+| INSTRUMENTATION_KEY              | This is the instrumentation key used by the app to talk to Application Insights                                                                                                         |No|
 
-Running the linting with auto fix:
+##### Additional test secrets
+Secrets required for getting tests to run correctly can be found in the below table. They are all accessible from the bootstrap keyvault.
 
-```bash
-$ yarn lint --fix
-```
+| Variable                     | Description                                |
+| ----------------------------| -------------------------------------------|
+| B2C_USERNAME                | User's username for B2C authentication     |
+| B2C_PASSWORD                | User's password for B2C authentication     |
+| B2C_ADMIN_USERNAME          | B2C administrator's username               |
+| B2C_ADMIN_PASSWORD          | B2C administrator's password               |
+| CFT_INVALID_USERNAME        | Invalid username for CFT authentication    |
+| CFT_VALID_USERNAME          | Valid username for CFT authentication      |
+| CFT_VALID_PASSWORD          | Valid password for CFT authentication      |
+| CFT_INVALID_PASSWORD        | Invalid password for CFT authentication    |
+| B2C_SYSTEM_ADMIN_USERNAME   | B2C system administrator's username        |
+| B2C_SYSTEM_ADMIN_PASSWORD   | B2C system administrator's password        |
```

Comment:
For the new codecept end2end tests, will also need the following env variables:

DATA_MANAGEMENT_URL
SUBSCRIPTION_MANAGEMENT_URL
SYSTEM_ADMIN_PROVENANCE_ID
VERIFIED_USER_ID